linear_feedback_controller_msgs_py/lfc_py_types.py

```diff
@@ -52,3 +54,4 @@ class Control:
     feedback_gain: npt.NDArray[np.float64]
     feedforward: npt.NDArray[np.float64]
     initial_state: Sensor
+    stamp: Time = Time()
```

Comment:
you need to use a factory function here like [in here](collision_pairs: list[Tuple[str, str]] = field(
        default_factory=list
    )  # list of collision pairs):
```python
collision_pairs: list[Tuple[str, str]] = field(
        default_factory=list
    )  # list of collision pairs
```

Reply:
Good catch. Done.